Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/pdf/core/DccExportFilter.kt

```diff
@@ -30,7 +29,7 @@ internal fun List<CwaCovidCertificate>.sort(): List<CwaCovidCertificate> = sorte
         { it.fullName },
         {
             when (it) {
-                is TestCertificate -> it.sampleCollectedAt?.toLocalDateUserTz()?.toJavaTime()
+                is TestCertificate -> it.sampleCollectedAt
```

Comment:
this is not going to work properly and may cause a crash 💥. 
The change you made compare an `Instant` to `LocaleDate`s. `vaccinatedOn` and `testedPositiveOn` are `LocaleDate`s while `sampleCollectedAt ` is an `Instant`

Reply:
if time is crucial in comparison as issue reports, maybe we need to convert all dates here into `Instant`s

Reply:
Converted all dates to instant

Reply:
looks good. lint is failing